### ISSUE TITLE
Docs: Interactivity API - Create and "Interactivity API" section under "How to Guides" section and include some initial guides

### ISSUE THREAD

**Issue by juanmaguitar** [WordPress/gutenberg#62922]
There are some guides to complement the docs at [Interactivity API Reference](https://developer.wordpress.org/block-editor/reference-guides/interactivity-api/) that could be added to a new "[How to Guides](https://developer.wordpress.org/block-editor/how-to-guides/) > Interactivity API" section

Some of the guides that could go under this new section are:
- [ ] How to Guides > Interactivity API
  - [ ] Adding interactivity to existing blocks
  - [ ] Using interactivity API in classic themes

**Comment by WPLake** [comment]
It would be nice to highlight the fact that iAPI can be used in any theme, including classic ones. Providing a small practical example would encourage developers to apply it even while developing classic themes.

Currently, it states that the API is public but does not provide a complete example. For instance, if developers try to use the `@wordpress/interactivity` alias directly, it won't work because WordPress doesn't define this alias automatically when it's used out of Gutenberg blocks.

Therefore, the example should describe the way to 'enable' the alias, or at least mention that the full root path (/wp-includes/...) is necessary for usage. Otherwise, many developers might feel that using iAPI outside of Gutenberg is not expected or encouraged.

**Comment by juanmaguitar** [comment]
> It would be nice to highlight the fact that iAPI can be used in any theme, including classic ones. Providing a small practical example would encourage developers to apply it even while developing classic themes.

@WPLake, this sounds like a great guide for the "How-to Guides > Interactivity API." Would you be up to writing this guide? 

If the guide becomes too long, or it turns out to more like a tutorial, it could be [suggested as](https://developer.wordpress.org/news/how-to-contribute/) a [Developer Blog post](https://developer.wordpress.org/news/) and just link it from "How-to Guides > Interactivity API."

**Comment by WPLake** [comment]
@juanmaguitar it seems more like a simple example with a description rather than a guide. This example is included in our article within [this section](https://wplake.org/blog/wordpress-interactivity-api/#54-in-plain-php-templates_wtl6). 

While writing the Interactivity API page, you can copy that part of the article and reword it to fit the flow of the page. The idea is to showcase that it's legimit to use in the classic themes too. Otherwise, if you copy-paste any example, it won't work due to the missing `@wordpress/interactivity` alias.
